cluster-install/install.sh

```diff
@@ -2273,8 +2275,8 @@ step_monitor_deployment_progress() {
 #======================================================================
 #== Main
 #======================================================================
-logln $__INFO "====== Astra Cluster Installer v0.0.1 ======"
 set_log_level
+logln $__INFO "====== Astra Cluster Installer v0.0.1 ======"
```

Comment:
Moved this line down as it was causing some buggy output due to the log level not being set yet.